tfjs-backend-webgpu/src/backend_webgpu.ts

```diff
@@ -574,6 +587,12 @@ export class WebGPUBackend extends KernelBackend {
           name: program.constructor.name,
           query: this.getQueryTime(this.querySet)
         });
```

Comment:
To simplify the code, maybe we can just remove the check of `this.supportTimeQuery` and do it like below:
```
     if (shouldTimeProgram) {
        this.activeTimers.push({
          name: program.constructor.name,
          query: this.getQueryTime(this.querySet)
        });
      }
```
And in getQueryTime, we just keep the original code
```
  async getQueryTime(query: GPUQuerySet): Promise<number> {
    if (this.supportTimeQuery) {
      return this.getTimeFromQuerySet(query);
    } else {
      return 0;
    }
  }
```
In this way, we don't need to add any cpuTimer and just return 0 if the user enables the profile mode with time query unsupported.

How do you think?